homeassistant/components/bluesound/media_player.py

```diff
@@ -206,20 +259,44 @@ async def async_service_handler(service: ServiceCall) -> None:
         )
 
 
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None,
+) -> None:
+    """Trigger import flows."""
+    setup_services(hass)
```

Comment:
Right, I see why we store the players in Hass data. We should move the setup of services to `async_setup` in `__init__.py`. i think for now you can keep the players in hass.data, just because this is here and the scope would be too big of this PR.

Check mealie for an example.

Reply:
But we should move this to async setup entry for now